RPS_ver2.py

```diff
@@ -38,22 +44,25 @@ def decide_winner(p1, p2):
 while True:
 
     # gather the player choices and convert to lowercase for easy comparison later
-    player1 = raw_input('Player one.. Rock, Paper, or Scissors? ').lower()
-    player2 = raw_input('Player two.. Rock, Paper, or Scissors? ').lower()
+    player_one_input = raw_input(rps_string.format("One")).lower()
+    player_two_input = raw_input(rps_string.format("Two")).lower()
 
     # If the test_input() returns
-    if test_input(player1) == True and test_input(player2) == True:
-        break
-
-    else:
+    if player_one_input not in SET_PLAYS or player_two_input not in SET_PLAYS:
+        # ghetto way to quit, but going for it.
         print 'Please enter a choice that is either Rock, Paper, or Scissors!'
+    else:
+        # Test prints for the chosen input
+        # print player1, type(player1)
+        # print player2, type(player2)
+
+        # Compute who is the winner
+        winner = decide_winner(player_one_input, player_two_input)
 
-# Test prints for the chosen input
-# print player1, type(player1)
-# print player2, type(player2)
+        # Who won?
+        print 'And the winner is... {0}!!!!'.format(winner)
```

Comment:
More of this formatting again.  Some people hate it, but I find it's easier not having to type " + " in between everything I want to do.